components/Navbar.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import { SafeAreaView, StatusBar, StyleSheet, Text, View, TouchableOpacity } from 'react-native';
+
+const Navbar = ({ onColorChange }) => {
+  const [activeIndex, setActiveIndex] = useState(null);
+
+  const colors = ['#FFD3FA', '#FFE2CC', '#FFF7CC', '#D3F2D7', '#CDF6FF'];
```

Comment:
Name these colors so they are easily readible, then you can still put them in array